.github/workflows/iroha2-dev-pr.yml

```
@@ -35,14 +35,38 @@ jobs:
       - name: Check Docker Compose configurations
         if: always()
         run: ./scripts/tests/consistency.sh docker-compose
+  
+  build_executor:
+    runs-on: [self-hosted, Linux, iroha2]
+    container:
+      image: hyperledger/iroha2-ci:nightly-2024-04-18
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v4
+      - uses: Swatinem/rust-cache@v2
+      - name: Build iroha executor
+        run: cargo run --bin iroha_wasm_builder_cli -- build ./default_executor --optimize --outfile configs/swarm/executor.wasm
```

```suggestion
        run: cargo run --release --bin iroha_wasm_builder_cli -- build ./default_executor --optimize --outfile configs/swarm/executor.wasm
```

should it run in release mode?

---

probably doesn't metter since it will build `iroha_wasm_builder`, actual executor always builds in release mode